File: medicalcare_frontend/src/app/models/account.ts
export class Account {
    constructor(
        public id: number,
        public email: string,
        public password: string,
        public first_name: string,
        public last_name: string,
        public dob: Date,
        public gender: string,
        public account_type: string,
        public address: string,
        public phone: string
    ){

    }
}

File: Medicalcare_API/Models/Account.cs

using System.ComponentModel.DataAnnotations;

namespace Medicalcare_API.Models{
    public class Account{
        
        [Key]
        public int id{get;set;}

        [Required]
        [MaxLength(50)]
        public string? email{get;set;}

        [Required]
        [MaxLength(50)]
        public string? password{get;set;}
        

        [Required]
        [MaxLength(50)]
        public string? first_name{get;set;}

        [Required]
        [MaxLength(50)]
        public string? last_name{get;set;}

        public DateTime? dob{get;set;}
        

        public string? gender{get;set;}

        public string? address{get;set;}
        

        public string? phone{get;set;}

        public string? account_type{get;set;}

        public string? token{get;set;}
    }
}